Spec/Auth.swift

```diff
@@ -2441,6 +2441,26 @@ class Auth : QuickSpec {
                     }
                 }
 
+                it("example: if is initialised with custom ttl and a TokenParams object is passed with null, then the @ttl@ used for every subsequent authorization will be null") {
+                    let options = AblyTests.commonAppSetup()
+                    options.defaultTokenParams = { $0.ttl = 0.1; return $0 }(ARTTokenParams())
+                    let rest = ARTRest(options: options)
+
+                    waitUntil(timeout: testTimeout) { done in
+                        rest.auth.authorise(nil, options: nil) { tokenDetails, error in
```

Comment:
Ignore my previous comment about not setting the TTL, in fact I do do that in the Ruby lib as well :/ https://github.com/ably/ably-ruby/blob/master/lib/ably/auth.rb#L34